game-app/game-core/src/main/java/games/strategy/triplea/ai/pro/ProNonCombatMoveAi.java

```diff
@@ -102,9 +102,13 @@ Map<Territory, ProTerritory> doNonCombatMove(
     final List<ProTerritory> prioritizedTerritories = prioritizeDefendOptions(factoryMoveMap);
 
     // Determine which territories to defend and how many units each one needs
-    final int enemyDistance =
-        ProUtils.getClosestEnemyLandTerritoryDistance(data, player, proData.getMyCapital());
-    moveUnitsToDefendTerritories(prioritizedTerritories, enemyDistance);
+    final Territory myCapital = proData.getMyCapital();
+    int enemyDistanceToMyCapital = Integer.MAX_VALUE;
+    if (myCapital != null) {
+      enemyDistanceToMyCapital =
+          ProUtils.getClosestEnemyLandTerritoryDistance(data, player, myCapital);
+      moveUnitsToDefendTerritories(prioritizedTerritories, enemyDistanceToMyCapital);
+    }
```

Comment:
Have you tested out this logic block?

Just wondering how the block below would behave with a large value of `enemyDistanceToMyCapital` would work below with:
```
defenseRange = enemyDistanceToMyCapital - 1;
```

Side-comment: I think it's a shame that `enemyDistanceToMyCapital` variable was used for such a large scope, notably 40 lines later.. quite a bit for a pretty innocuous looking local variable.

Reply:
I believe the block below is inside a capital is not null check, so wouldn't be hit in this case.